components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/elements/definitions/stream.js

```diff
@@ -44,10 +44,16 @@ define(['require', 'elementUtils'],
                 this.previousCommentSegment = options.previousCommentSegment;
                 this.name = options.name;
             }
+
+            this.annotationListObjects = [];
             this.attributeList = [];
             this.annotationList = [];
```

Comment:
do we need this annotationList now ?

Reply:
yes we need as because still the backend is based on the annotationList(string annotations)